app/src/org/commcare/activities/FormEntryActivityUIController.java

```diff
@@ -498,8 +498,14 @@ private void createRepeatDialog() {
 
         // Assign title and text strings based on the current state
         String backText = Localization.get("repeat.dialog.go.back");
-        String addAnotherText = repeatCaptionPrompt.getRepeatText(FormEntryActivity.mFormController.getLastRepeatCount() > 0 ? "add" : "add-empty");
+
+        // Checking the repetitions here
```

Comment:
It's a bad practice to introduce self explanatory comments in the code and comments should only be used when needed due to a counter-intuitive or complex piece of code.

Reply:
Done